uaa_client/authentication.py

```diff
@@ -154,5 +154,8 @@ def authenticate(self, uaa_oauth2_code=None, request=None, **kwargs):
             return None
 
         user_info = jwt.decode(access_token, verify=False)
+        email = user_info['email']
 
-        return self.get_user_by_email(user_info['email'])
+        logger.info('Authenticating user with email {}'.format(email))
```

Comment:
Note that this doesn't necessarily log absolute success; the login can still ultimately fail if the following `self.get_user_by_email()` call fails, but that method will log something if it fails.